extensions/Khronos/KHR_mesh_compression/README.md

```diff
@@ -0,0 +1,172 @@
+# KHR_draco_geometry_compression
```

Comment:
This extension is proposed by Google, so should it use `GOOGLE` (or `EXT`) prefix instead of `KHR`?
Also, since it extends `mesh` object, maybe it should be called `_mesh_draco_compression` to be aligned with other extension names?